.github/vale/styles/Google/Periods.yml

```diff
@@ -0,0 +1,7 @@
+extends: existence
+message: "Don't use periods with acronyms or initialisms such as '%s'."
```

Comment:
I assume this takes into account and omit all the words defined in the dictionaries (such as API)?

Reply:
Oh, sorry, I should have added that in the PR description. The files in `/Google` are out generated from the package, so it's nothing we can (or should) edit. 

And yes, our Klaw vocabulary is higher prio then this. 

I think in time we'll establish our own rules more, at that point we may don't even need the google package anymore. I thought it's a good baseline for us.